dace/runtime/include/dace/xilinx/access.h

```diff
@@ -71,4 +71,24 @@ void Write(StreamView<T, vector_length, capacity> &stream_view,
   stream_view.push(value);
 }
 
+template <typename T, unsigned vector_length>
```

Comment:
isn't this redundant with the opencl-generated pack/unpack?

Reply:
This is Xilinx, vectors are defined completely differently, so we don't use the same functions